android/app/build.gradle

```diff
@@ -39,8 +39,9 @@ android {
 
     defaultConfig {
         applicationId "com.olmps.memoClient"
-        minSdkVersion 16
+        minSdkVersion 17
         targetSdkVersion 30
+        multiDexEnabled true
```

Comment:
`quill_version` now depends on `youtube_player_flutter` that depends on `flutter_inapp_webview` that [requires](https://github.com/pichillilorenzo/flutter_inappwebview/blob/f06bcdf695847a0187a31ab607b2bfe236fb0057/android/build.gradle#L28) min sdk version 17 and `multiDexEnabled` enabled